### PR TITLE
Update macOS instructions

### PR DESCRIPTION
**Pull request by qwertychouskie**
Thanks @OzoneOne for testing this.

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -204,10 +204,10 @@ make
 By default, the executable that is produced is not ready for distribution. Install https://github.com/auriamg/macdylibbundler
 
 ```bash
-dylibbundler -od -b -x ./bin/SuperTuxKart.app/Contents/MacOS/supertuxkart -d ./bin/SuperTuxKart.app/Contents/libs/ -p @executable_path/../libs/
+dylibbundler -od -b -x ./bin/SuperTuxKart.app/Contents/MacOS/supertuxkart -d ./bin/SuperTuxKart.app/Contents/libs/ -p /usr/local/opt/
```
Comment (auriamg) [inline INSTALL.md]: This is wrong, this change will make the executable impossible to distribute

File: INSTALL.md [hunk 2]
```diff
@@ -236,6 +236,16 @@ Building on 10.10 with 10.9 compatibility:
 cmake .. -DCMAKE_OSX_SYSROOT=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.9.sdk  -DCMAKE_OSX_DEPLOYMENT_TARGET=10.9
 ```
 
+#### (Optional) packaging for distribution
```
Comment (auriamg) [inline INSTALL.md]: 0.9.3 does not need this.

Comment (qwertychouskie) [reply INSTALL.md]: Removed.

File: INSTALL.md (hunk 1 above)
Comment (qwertychouskie) [reply INSTALL.md]: Fixed.

File: INSTALL.md [hunk 3]
```diff
@@ -235,19 +235,3 @@ Building on 10.10 with 10.9 compatibility:
 ```bash
 cmake .. -DCMAKE_OSX_SYSROOT=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.9.sdk  -DCMAKE_OSX_DEPLOYMENT_TARGET=10.9
 ```
-
```
Comment (auriamg) [inline INSTALL.md]: Why did you remove parts of the 0.9.3 instructions? Just leave them in place please until the new version is released
